python/features/plugins/print_defs.feature

```diff
@@ -1,9 +1,9 @@
 Feature:  Print-Defs Evaluation
 
     Scenario: Print Definitions from Context
-        When "print-defs" is called with args " " and flags " "
+        When I run the "print-defs" command with no arguments or flags
```

Comment:
Shouldn't this be
```suggestion
        When I run the "print-defs" command with no arguments and with no flags
```

Reply:
So, understand now these are worded to match what you have in the steps file. Believe we should stay consistent in our phrasing for the test though.